frontend/src/Components/ReviewsModal/ReviewsModal.css

```diff
@@ -0,0 +1,59 @@
+.modal {
+  display: block;
+  position: fixed;
+  z-index: 1;
+  padding-top: 60px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
```

Comment:
nit: duplicates. i think the lower one wins?

Reply:
You're right